.gitlab-ci.yml

```diff
@@ -32,7 +32,7 @@ variables:
   GIT_STRATEGY: fetch
   GIT_DEPTH: 100
   CI_SERVER_NAME: "GitLab CI"
-  CI_IMAGE: "paritytech/ci-linux:production"
+  CI_IMAGE: "paritytech/ci-linux@sha256:fdde968a13206e169bc98f905f76640aeddd5abb3e057381e6eccc178b9f5277" # staging 2023-03-16
```

Comment:
staging?

Reply:
yep, that version will be promoted to production once our repos are confirmed to build with it